### PR TITLE
OSDOCS-11383: GCP Workload ID webhook RN

### PR DESCRIPTION
**Pull request by jeana-redhat**
:rotating_light: DO NOT MERGE :rotating_light:

:arrow_right: PR for SME acks and peer review

Link to docs preview:
[Authenticating customer workloads with GCP Workload Identity](https://84541--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-4-enhancements-nodes-pods-short-term-auth_release-notes)

Reviews:
- [x] Dev has approved this change.
- [ ] QE has approved this change.
- [x] Peer review complete.

Change management acks to be collected on base PR #83682

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -2886,6 +2886,53 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+[id="ocp-4-17-4-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-17-4-enhancements-nodes-pods-short-term-auth_{context}"]
+===== Authenticating pods with {gcp-wid-short}
+
+With this release, applications in pods on {product-title} clusters that use {cloud-auth-first} can authenticate by using {cloud-auth-short}.
+
+To use this authentication method with your applications, you must complete configuration steps on the cloud provider console and your {product-title} cluster.
+For more information, see xref:../nodes/pods/nodes-pods-short-term-auth.adoc#nodes-pods-short-term-auth-configuring-gcp_nodes-pods-short-term-auth[Configuring GCP Workload Identity authentication for pods on GCP]
```
Comment (jeana-redhat) [inline release_notes/ocp-4-17-release-notes.adoc:2921]: This xref relies on #83682, which will merge before 4.17.4 RNs.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -2886,6 +2886,53 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+[id="ocp-4-17-4-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-17-4-enhancements-nodes-pods-short-term-auth_{context}"]
+===== Authenticating pods with {gcp-wid-short}
+
+With this release, applications in pods on {product-title} clusters that use {cloud-auth-first} can authenticate by using {cloud-auth-short}.
+
+To use this authentication method with your applications, you must complete configuration steps on the cloud provider console and your {product-title} cluster.
+For more information, see xref:../nodes/pods/nodes-pods-short-term-auth.adoc#nodes-pods-short-term-auth-configuring-gcp_nodes-pods-short-term-auth[Configuring GCP Workload Identity authentication for pods on GCP].
```
Comment (jeana-redhat) [inline release_notes/ocp-4-17-release-notes.adoc:2921]: This xref relies on #83682, which will merge before 4.17.4 RNs.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -2855,6 +2855,53 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[id="ocp-4-17-4-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-17-4-enhancements-nodes-pods-short-term-auth_{context}"]
+===== Authenticating customer workloads with {gcp-wid-short}
+
+With this release, applications in customer workloads on {product-title} clusters that use {gcp-wid-first} can authenticate by using {gcp-wid-short}.
+
+To use this authentication method with your applications, you must complete configuration steps on the cloud provider console and your {product-title} cluster.
```
Comment (lahinson) [inline release_notes/ocp-4-17-release-notes.adoc:2889]:
```suggestion
To use this authentication method with your applications, you must configure the cloud provider console and your {product-title} cluster.
```
Would "configure" mean the same thing as "complete configuration steps" here?

**Review by lahinson — COMMENTED** [review]
Looks good! Just one nit/question that you can take or leave at your discretion.

File: release_notes/ocp-4-17-release-notes.adoc (hunk 3 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-17-release-notes.adoc:2889]: thanks! they are configuring IAM things by using the console, but the full docs get into that detail

**Comment by jeana-redhat** [comment]
Changes merged in #84883

**Pull request closed without merging**